README.md

```diff
@@ -1,9 +1,55 @@
 # Gruntwork Installer
 
-At [Gruntwork](http://www.gruntwork.io/), we've developed a number of scripts and binaries, most of them in private
+`gruntwork-install` is a bash script you run to easily download and install "Script Modules" written by Gruntwork. 
+
+### Script Modules
+
+A Script Module is a package of one or more bash scripts and/or binaries maintained by Gruntwork that are meant to be 
+reusable in many different contexts. For example, we have Script Modules for installing a CloudWatch Logs agent, optimizing
+syslog settings, setting up automatic security updates, and more. 
+
+Our Script Modules are contained in the `modules/` folder of different GitHub repos since they often come with the 
+different [Infrastructure Packages](https://blog.gruntwork.io/gruntwork-infrastructure-packages-7434dc77d0b1#.6bwor6wxc) 
+we sell.
```

Comment:
I'm not sure if "Script Modules" is the right term. `gruntwork-install` now supports a `binary-name` parameter that installs an arbitrary binary uploaded to a GitHub release in the repo. This doesn't mesh well with the description above because:

1. We don't usually call binaries "scripts". 
1. The code isn't being installed from a `modules/` folder but a release. Granted, the code underneath that binary is probably in a `modules/` folder somewhere in that repo, but it's an important distinction.

Perhaps Executable Modules? Runnable Modules? Or, better yet, Gruntwork Apps? That is, the terminology is that you use `gruntwork-install` to install our Apps, which may be scripts or binaries used in a variety of contexts.